src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,70 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { useDispatch } from 'react-redux';
+import { actions } from '../../features/currentTodo';
+
+type Props = {
+  selectedTodo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    getUser(selectedTodo.userId).then(userFromServer => {
+      setUser(userFromServer);
+      setIsLoading(false);
```

Comment:
use `finally` for this to guarantee to close the loader and avoid infinity loading